examples/react-native/README.md

```diff
@@ -50,7 +50,7 @@ yarn react-native-example ios
 yarn && yarn build
 ```
 
-1. Build and install the Example App:
```

Comment:
In markdown shouldn't these automatically be getting converted to the sequential numbers? Is it not working because of the text between?

Reply:
It would if there're no interruptions, but there are code blocks in between 🥴

Reply:
Here's how it looks
https://github.com/aws-amplify/amplify-ui/blob/1f24933870a65836162534c6bf00d26e1d775158/examples/react-native/README.md